vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -487,6 +493,12 @@
       <DependentUpon>IReactNotificationService.idl</DependentUpon>
       <SubType>Code</SubType>
     </ClCompile>
+    <ClCompile Include="JsiReader.cpp">
```

Comment:
><ClC [](start = 4, length = 4)

It seems that the PR is missing changes to the MS.RN.vcsproj.filters file.

Reply:
Visual Studio doesn't change the filters file, the reason could be that, all added types have already been made depending on idl files.